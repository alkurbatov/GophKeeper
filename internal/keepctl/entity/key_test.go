package entity_test

import (
	"testing"

	"github.com/alkurbatov/goph-keeper/internal/keepctl/entity"
	"github.com/alkurbatov/goph-keeper/internal/libraries/gophtest"
	"github.com/gkampitakis/go-snaps/snaps"
)

func TestKeyToHash(t *testing.T) {
	sat := entity.NewKey(gophtest.Username, gophtest.Password)

	snaps.MatchSnapshot(t, sat.Hash())
}
