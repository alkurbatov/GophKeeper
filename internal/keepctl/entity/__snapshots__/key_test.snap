
[TestKeyToHash - 1]
88bb5abaa61568b9f11ba091445d81772a3a264fb3f3054088f78baf7a091a9d
---
